plugins/dotnet-auth-core-identity/src/index.ts

```diff
@@ -106,10 +103,16 @@ class AuthCorePlugin implements dotnetTypes.AmplicationPlugin {
       destPath,
       filePath,
       context,
-      CsharpSupport.classReference({
-        name: `${resourceName}`,
-        namespace: `${resourceName}.Infrastructure`,
-      })
+      [
+        CsharpSupport.classReference({
+          name: `${resourceName}`,
+          namespace: `${resourceName}.Infrastructure`,
+        }),
+        CsharpSupport.classReference({
+          name: `${resourceName}`,
+          namespace: `${resourceName}.Infrastructure.Models`,
+        }),
```

Comment:
Probably it's not needed anymore due to the fact that ProgramAuthExtensions won't reference the authEntity anymore

Reply:
@overbit  good catch :), removed